config/configmap/inferenceservice.yaml

```diff
@@ -40,9 +40,9 @@ data:
             "multiModelServer": "false"
         },
         "pytorch": {
-            "image": "gcr.io/kfserving/pytorchserver",
-            "defaultImageVersion": "v0.4.1",
-            "defaultGpuImageVersion": "v0.4.1-gpu",
+            "image": "pytorch/torchserve",
+            "defaultImageVersion": "latest",
```

Comment:
should always pin to a specific version

Reply:
We are working on an image with an envelope for KFServing, will update once we are done.

Reply:
Can use `kfserving/torchserve-kfs` for now